src/mem/request.hh

```diff
@@ -654,7 +654,7 @@ class Request
 
     /** Accesssor for the requestor id. */
     MasterID
-    masterId() const
+    uniqueId() const
```

Comment:
I think this should be "requestorId"

Reply:
If I name the function here requestorId, it'll have to be called that everywhere else, so I thought uniqueId made more sense as the MasterID class is now UniqueID.

Otherwise it would be an easy fix if you prefer requestorId()